src/main/java/com/study/platform/blog/FeedController.java

```diff
@@ -0,0 +1,30 @@
+package com.study.platform.blog;
+
+import com.study.platform.blog.service.FeedApiService;
+import com.study.platform.blog.service.dto.FeedDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+public class FeedController {
```

Comment:
```suggestion
@RequiredArgsConstructor
@RestController
@RequestMapping("/feeds")
public class FeedController {
```
feedController는 별일 없는 이상 feeds가 prefix로 붙을거 같아서 이렇게 처리하는게 좋겠네요.;

Reply:
저는 엔드 포인트는 명시적으로 풀네임을 쓰는 것도 좋다고 생각합니다 !